### ISSUE TITLE
[Booking] Add a "mandatory" option to Booking summary options and add email to these options

### ISSUE THREAD

**Issue by gaetanHerve** [bgo-bioimagerie/platformmanager#407]
Request coming from Alain-Hervé Le Gall, OSUR Rennes space

**Comment by osallou** [comment]
Why email, we have the user...

**Comment by gaetanHerve** [comment]
I think it's to get all informations needed in the same spot. The real need here probably is to be able to get a list of reservations (with all infos) for a resource and a period.

**Comment by ahlegall** [comment]
right: I got about 700 users... It would be easier to have all the infos in a glance
